tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -73,18 +78,24 @@
 import static io.dockstore.tooltester.helper.ExceptionHandler.IO_ERROR;
 import static io.dockstore.tooltester.helper.ExceptionHandler.exceptionMessage;
 import static io.dockstore.tooltester.helper.JenkinsHelper.buildName;
+import static io.dockstore.tooltester.runWorkflow.WorkflowRunner.printLine;
+import static java.lang.Thread.sleep;
 
 /**
  * Prototype for testing service
  */
 public class Client {
     private static final Logger LOG = LoggerFactory.getLogger(Client.class);
+    public static final int WAIT_TIME = 10000;
```

Comment:
Don't make it public if it doesn't need to be (I don't think it needs to be).

Reply:
Fixed